docs/decex/xdm.md

```diff
@@ -262,16 +262,17 @@ Detailed description of each function required to be present in the protocol.
 
 Consensus chain maintains a `ChainAllowList` to keep track of the authorized domain chains that can establish channels with the Consensus chain. `ChainAllowList` can be defined at Genesis and can be updated later by the sudo account adding or removing chains.
 
-When a Consensus chain receives an `initiate_channel` XDM to open the channel, if the `src_chain` is in the allowlist, then channel is opened else XDM is rejected.
+When a Consensus chain receives an `initiate_channel` XDM to open the channel, if the `dst_chain_id` is in the allowlist, then channel is opened else XDM is rejected.
```

Comment:
XDM wont be rejected but instead chain will not open the channel. It will stay in channel init state on both chains.
Once the channel is closed, channel owner can claim their reserve